hardware_interface/include/hardware_interface/actuator_interface.hpp

```diff
@@ -118,11 +118,13 @@ class ActuatorInterface : public rclcpp_lifecycle::node_interfaces::LifecycleNod
    * \param[in] hardware_info structure with data from URDF.
    * \param[in] clock pointer to the resource manager clock.
    * \param[in] logger Logger for the hardware component.
+   * \param[in] executor weak pointer to the MultiThreaderExecutor used by the controller manager.
```

Comment:
```suggestion
   * \param[in] executor weak pointer to the Executor used by the controller manager.
   * \warning The parsed executor should not be used to call `cancel()` or use blocking callbacks such as `spin()`.
```
THe executor need not be a MultiThreaderExecutor, as it is a shared pointer it can be anything

Reply:
I've added some docs to this

Reply:
Makes sense